### PR TITLE
Bump API to 1.0 + doc improvements

### PR DESCRIPTION
**Pull request by Kartones**

### CODE REVIEW

**Comment by Kartones** [comment]
@eferro CR please

File: README.md [hunk 1]
```diff
@@ -50,16 +59,40 @@ The API is hosted at the <strong>api.themotion.com</strong> domain.
...
+                  +----->|              |
+                         +--------------+
+```
+
+### Batch Creation Flow
+
+We recommend the following course of action when creating a batch in order to reduce both complexity of client integrations/
+
+1. Create the batch by calling the `Batch Creation` endpoint
+2. Setup in your system URLs that allow public access, in order to receive callbacks of results and/or errors
```
Comment (eferro) [inline README.md]: Perhaps this is the step 1 and create the batch is the second one...

File: README.md [hunk 2]
```diff
@@ -227,6 +260,7 @@ Request data example:
         <tr><td>template.id</td><td>yes</td><td>String</td><td>This field must be provided in case the client/consumer want to use a special template. Otherwise a default template will be used to produce the videos.</td></tr>
         <tr><td>audio_track.id</td><td>no</td><td>String</td><td>This field must be provided in case the client/consumer want to use a special audio track. Otherwise will be generated muted.</td></tr>
         <tr><td>video.resolution</td><td>yes</td><td>String</td><td>This is the list of the desired video sizes, for instance "1920x1080", Please, refer to the <a href="#templates-available-filtered-by-ids">Templates Available Filtered By Ids</a> in order to obtain the available resolutions for a particular Template</td></tr>
+        <tr><td>email</td><td>no</td><td>String</td><td>If given a valid email address, will send an email when the batch finishes containing the thumbnail and a link to of each of the generated videos. Not recommended for big batches.</td></tr>
```
Comment (eferro) [inline README.md]: Only a question, why is not recommended for big batches? is a requirement from business?  I have no idea about this...

Comment (Kartones) [inline README.md]: Because the email will get huge, but I can add a comment with the reason

**Comment by eferro** [comment]
+1

**Pull request merged**